exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = roomMaxCapacity != enoughSpaceInRoom; // finish this statement
 
```

Comment:
Hi Bimbola, In here you need to show if there is enough space in room for students and mentors, in other words you need to assign a value to enoughSpaceInRoom variable that shows if you have enough room for all the students and mentors. for example if max capacity of the room was more than or equal to the combined number of students and mentors then you will have enough room.